examples/select-file.c

```diff
@@ -4,7 +4,7 @@
   contents of a selected file.
 */
 
-#include_shell "posix-utils.sh"
+#include_shell "examples/posix-utils.sh"
```

Comment:
For `#include`, pnut opens files relative to the current directory of the file and I think `#include_shell` should do the same. I made https://github.com/udem-dlteam/pnut/pull/119 to fix this small bug.

Reply:
Agreed.  I'll drop that patch once your other change is in and I can rebase this pull request.